### PR TITLE
Initialize the structure for frontend part

### PR DESCRIPTION
**Pull request by jeffrey0706**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,30 @@
...
+# # misc
+# .DS_Store
+# .env.local
+# .env.development.local
+# .env.test.local
+# .env.production.local
+
+# npm-debug.log*
+# yarn-debug.log*
+# yarn-error.log*
```
Comment (jeffrey0706) [inline .gitignore:51]: These are kept in case we need them later

File: frontend/README.md
Comment (jeffrey0706) [inline frontend/README.md:1]: This file is auto-generated by react

Comment (ander1119) [reply frontend/README.md:1]: should we create README for frontend? and the README in root folder only for integration test

Comment (jeffrey0706) [reply frontend/README.md:1]: It's already here= =

**Review by ander1119 — APPROVED** [review]

**Pull request merged**